google/cloud/storage/_signing.py

```diff
@@ -91,7 +91,9 @@ def get_expiration_seconds_v2(expiration):
     """Convert 'expiration' to a number of seconds in the future.
 
     :type expiration: Union[Integer, datetime.datetime, datetime.timedelta]
-    :param expiration: Point in time when the signed URL should expire.
+    :param expiration: Point in time when the signed URL should expire. If
+                       expiration in datetime, then need to pass timezone
+                       with it. Default value of ``tzinfo`` is ``UTC``.
```

Comment:
The default value of the `tzinfo` argument to `datetime.datetime` is None, resulting in a "naive" `datetime` instance:  the comment should state clearly that we forcibly convert such instances to `UTC`.  E.g.:
```python
    :param expiration: Point in time when the signed URL should expire. If
                       a `datetime` instance is passed without an explicit
                      `tzinfo` set,  it will be converted to ``UTC``.
```

Reply:
Rather than "it will be converted to" let's use "it will be assumed to be". This is a huge footgun; a user who sees the term "converted" will assume that we will do the right thing in the default case, whereas in fact we will almost always do the wrong thing.